data/moves.js

```diff
@@ -3120,6 +3120,7 @@ let BattleMovedex = {
 			onTryHitPriority: 3,
 			onTryHit(target, source, move) {
 				if (move && (move.target === 'self' || move.category !== 'Status')) return;
+				if (move.id === 'perishsong') return;
```

Comment:
```suggestion
				if (['self', 'all'].includes(move.target) || move.category !== 'Status') return;
```

(`move.target === 'all'` covers Perish Song.)